drivers/hmis/app/graphql/types/hmis_schema/custom_data_element.rb

```diff
@@ -41,5 +41,9 @@ def values(parent:)
     def value(parent:)
       object.repeats ? nil : all_values(parent: parent)&.first
     end
+
+    def activity_log_object_identity
+      object.id
```

Comment:
The resolver combines the parent id for some reason. I assume it's okay to use the record id for logging